### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Mmeo0** [MadeBaruna/paimon-moe#290]
![изображение_2022-08-10_145259546](https://user-images.githubusercontent.com/110985050/183906053-3d388e8b-5f53-4e41-94c3-b8c8affffc65.png)
where is Xiaos banner ? i did 350+ pulls on his (characters) and jade spear (weapon) banner during winter, but i cant see it there. theres also no xiaos rerun. fix it pls :)

**Comment by Ilya0S** [comment]
Based on what I see, they're combined. I haven't pulled for Kazuha, but I did for Klee, and I have that amount listed for Kazuha.
I guess it auto-assigns based on time pulled, so using double markings (e.g. Kazuha / Klee) would somewhat fix that.

**Comment by Mmeo0** [comment]
> Based on what I see, they're combined. I haven't pulled for Kazuha, but I did for Klee, and I have that amount listed for Kazuha. I guess it auto-assigns based on time pulled, so using double markings (e.g. Kazuha / Klee) would somewhat fix that.

Yeah, i see. But if u look at the screenshot , u will see shenhe's banner, which was at the same time as xiao's (5.1.2022). i pulled for xiao (350+ wishes) , but not shenhe. if it was combined, id be able to see those 350+ pulls on shenhes banner but i cant. theres also NO info abt xiaos weapon banner.
